native_client/client.js

```diff
@@ -0,0 +1,14 @@
+const Fs = require('fs');
+const Sox = require('sox-stream');
+const Ds = require('./javascript/build/Release/deepspeech');
```

Comment:
The installation of a module built in this way is a bit unclear, so it depends entirely on where you decide to put files I suppose. I think this will have to do for now - I'm sure it can be refined, but I think most node users understand that require can take a path. You don't really 'install' modules like you do with Python (it's possible, but discouraged), they're all local to the source tree.